features/support/legacy_web_steps_support.rb

```diff
@@ -283,7 +283,7 @@ def path_to(page_name)
 end
 
 World(NavigationHelpers)
-EOF
+    EOF
```

Comment:
This, doesn't it require the squiggly heredoc `<<~EOF` at its start?

Reply:
So this entire file is 2 massive heredocs.

Heredoc 1 is: 3-250
Heredoc 2 is: 252-286

I've not tweaked anything here other than a tab, and it still all runs. However this entire file / set of steps should be just deleted because they're so bad. Aslak has put a massive comment at the top eluding to this.